mozci/task.py

```diff
@@ -218,6 +222,14 @@ def to_json(self):
         sig = signature(self.__init__)
         return {k: v for k, v in self.__dict__.items() if k in sig.parameters}
 
+    def retrigger(self):
+        new_task_id = taskcluster.slugId()
```

Comment:
I think we'll need to implement this `_should_retrigger` check here too:
https://searchfox.org/mozilla-central/source/taskcluster/gecko_taskgraph/actions/retrigger.py#160

Some tasks could cause problems if they get retriggered, so they have a way of declaring that they do not. This check acts as a safeguard to prevent accidentally retriggering them when we shouldn't.